src/application/preRequests/writeOperationPreRequestScripts.ts

```diff
@@ -22,12 +23,19 @@ export const writeOperationPreRequestScripts = (
       } as EventDefinition
     }
 
+    const scriptContents = scripts.map(src => {
```

Comment:
Same remark, it would be safer to wrap this in a try {} catch, and throw an error if the file does not exist.